docs/index.rst

```diff
@@ -51,14 +51,16 @@ focus_offset  SIM focus offset [steps] (default=0)
 
 **Optional**
 
-============== =========================================================
-Argument       Description
-============== =========================================================
-include_ids    list of AGASC IDs of stars to include in selected catalog
-include_halfws list of acq halfwidths corresponding to ``include_ids``
-exclude_ids    list of AGASC IDs of stars to exclude from selected catalog
-stars          table of AGASC stars (will be fetched from agasc if None)
-============== =========================================================
+=================== =========================================================
+Argument            Description
+=================== =========================================================
+include_ids_acq     list of AGASC IDs of stars to include in acq catalog
+include_halfws_acq  list of acq halfwidths corresponding to ``include_ids``
```

Comment:
The '_acq' seems a little odd for the halfw (because we don't handle halfw for guide or fid), but I suppose it is consistent.